docs/content/documentation/modules/index.md

```diff
@@ -2,19 +2,80 @@
 
 MOOSE includes a set of community developed physics modules that you can build on to create your own application.
 
-* [Phase Field](modules/phase_field/index.md)
-* [Tensor Mechanics](modules/tensor_mechanics/index.md)
-* [Reconstructed Discontinous Galerkin](modules/rdg/index.md)
-* [Navier-Stokes](modules/navier_stokes/index.md)
-* [Level Set](modules/level_set/index.md)
-* [Fluid Properties](modules/fluid_properties/index.md)
-* [Stochastic Tools](modules/stochastic_tools/index.md)
-* [Porous Flow](modules/porous_flow/index.md)
-* [Chemical reactions](modules/chemical_reactions/index.md)
+- [Phase Field](modules/phase_field/index.md)
+- [Tensor Mechanics](modules/tensor_mechanics/index.md)
+- [Reconstructed Discontinous Galerkin](modules/rdg/index.md)
+- [Navier-Stokes](modules/navier_stokes/index.md)
+- [Level Set](modules/level_set/index.md)
+- [Fluid Properties](modules/fluid_properties/index.md)
+- [Stochastic Tools](modules/stochastic_tools/index.md)
+- [Porous Flow](modules/porous_flow/index.md)
+- [Chemical reactions](modules/chemical_reactions/index.md)
 
 The purpose of the modules is to encapsulate common kernels, boundary conditions, etc. to prevent code duplication.
 Examples include: heat conduction, solid mechanics, Navier-Stokes, and others. The modules are organized so that your
 application can link against only those which it requires.
 
 !!! warning
     _No_ export controlled physics (e.g., neutronics) should be put into the modules.
+
+## Modules Anatomy
+---
```

Comment:
How are your find and replace skills? I would like all these "---" to be removed. If we want a horizontal line we should do it with css rather than making manual lines like this.

Reply:
@aeslaughter How do we do this in CSS, in markdown? I was looking for an example, but I can't find one in our markdown style guide page.

Reply:
I don't think we need the lines at all, I wouldn't worry about it.